mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/MavenProjectConfigurator.java

```diff
@@ -99,36 +98,10 @@ private void addSkipAfterFailureCount(PluginExecution pluginExecution) {
         }
 
         return testRunnerPluginConfigurations.stream()
-            .filter(
-                testRunnerPlugin -> !(testRunnerPlugin.getArtifactId().equals("maven-failsafe-plugin") && isSkipITs()))
+            .filter(testRunnerPlugin -> !(isFailsafePlugin(testRunnerPlugin) && isSkipITs()))
```

Comment:
Maybe extract filter condition to meaningful method - would be easier to understand the `why` of filtering